kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ConfigurationServiceImpl.java

```diff
@@ -207,11 +209,10 @@ public ConfigurationServiceImpl() {
     //
     // ----------------------------------------------------------------
 
-    protected void activate(ComponentContext componentContext) throws InvalidSyntaxException {
+    protected void activate() throws InvalidSyntaxException {
         s_logger.info("activate...");
 
-        // save the bundle context
-        this.m_ctx = componentContext;
+        this.bundleContext = FrameworkUtil.getBundle(ConfigurationServiceImpl.class).getBundleContext();
```

Comment:
As we are using R6, we can use 

`void activate(ComponentContext,  BundleContext,  Map<String,Object>)`